src/app/tests/TestCommandInteraction.cpp

```diff
@@ -1196,6 +1203,52 @@ void TestCommandInteraction::TestCommandHandlerWithProcessReceivedEmptyDataMsg(n
     }
 }
 
+void TestCommandInteraction::TestCommandSenderLegacyUnsupportedCommand(nlTestSuite * apSuite, void * apContext)
```

Comment:
Need a case to handle having N command paths, but not a per-path-handling-callback. The SendCommandRequest call should fail.

Reply:
Please note in the PR description:

> Things to be completed in a follow up:
> 
> CommandSender sets mBatchCommandsEnabled to true if build config CHIP_CONFIG_SENDING_BATCH_COMMANDS_ENABLED is enabled and client sets the max number of paths remote peer supports to a value > 1. Once this is reworked to have the callers intention to batch more explicitly make their intentions knows that they will be sending multiple commands in a single invoke, we will use the new version of mClientPlansToSendBatchCommands to enforce the correct return value from UsingExtendedPathCallbacks being true.